modules/ROOT/pages/customer-connector-upgrade.adoc

```diff
@@ -0,0 +1,162 @@
+= Customer Guide for Upgrading and Testing Custom Connectors for Java 17
+
+Upgrade and test your custom connectors for Java 17 to ensure compatibility within the MuleSoft ecosystem. 
+
+== Before You Begin
+
+Before upgrading and testing your custom connector for Java 17, you must be familiar with the following:
+
+* *How to run and compile your custom connector*
```

Comment:
So, any text in bold will not be translated. Our guidance is to not use bold for emphasis. If you want it to stick out, consider using a DL.

Reply:
Also, think about how to make these bullets parallel.
Should this be something like "Target Compilation Level"?